tests/h/search/query_test.py

```diff
@@ -192,6 +192,46 @@ def test_it_sorts_annotations(self, Annotation, search, sort_key, order, expecte
     def test_it_ignores_unknown_sort_fields(self, search):
         search.run({"sort": "no_such_field"})
 
+    @pytest.mark.parametrize("date,expected",
+        [("1514773561300", [2]),
+         ("2018-01-01T02:26:01.03", [2]),
+         ("2018-01-01T02:26:01.03+00:00", [2]),
+         ("2018-01-01", [1, 2])],
+    )
+    def test_it_finds_all_annotations_after_ms_since_epoch_date(
```

Comment:
This function name needs updating to reflect the fact that the inputs are a mix of formats, not only ms-since-epoch.